bpdm-pool/src/main/kotlin/org/eclipse/tractusx/bpdm/pool/service/RequestValidationService.kt

```diff
@@ -225,6 +257,29 @@ class RequestValidationService(
         }.groupBy({ it.first }, { it.second })
     }
 
+
+    fun findDuplicateLegalEntityIdentifiers(
+        legalEntityRequests: List<LegalEntityDto>
+    ): Set<LegalEntityIdentifierDto> {
+        val allIdentifiers = legalEntityRequests.flatMap { it.identifiers }
+
+        return allIdentifiers.groupBy { it }
```

Comment:
This should only consider the fields `type` and `value`. Maybe we could use data class `IdentifierCandidateKey` for that?